test/instrumentation/stability-utils.sh

```diff
@@ -121,7 +121,9 @@ kube::update::documentation::list() {
 kube::update::documentation() {
   stability_check_setup
   temp_file=$(mktemp)
-  doUpdateDocs=$(go run "test/instrumentation/documentation/main.go" -- 1>"${temp_file}")
+  arg1=$1
+  arg2=$2
+  doUpdateDocs=$(go run "test/instrumentation/documentation/main.go" --major $arg1 --minor $arg2 -- 1>"${temp_file}")
```

Comment:
```suggestion
  doUpdateDocs=$(go run "test/instrumentation/documentation/main.go" --major "$arg1" --minor "$arg2" -- 1>"${temp_file}")
```

According to test failure , https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/113610/pull-kubernetes-verify/1588289616308342784

/hold
to avoid auto-retest